README.md

```diff
@@ -1,9 +1,55 @@
 # Gruntwork Installer
 
-At [Gruntwork](http://www.gruntwork.io/), we've developed a number of scripts and binaries, most of them in private
+`gruntwork-install` is a bash script you run to easily download and install "Script Modules" written by Gruntwork. 
+
+### Script Modules
+
+A Script Module is a package of one or more bash scripts and/or binaries maintained by Gruntwork that are meant to be 
+reusable in many different contexts. For example, we have Script Modules for installing a CloudWatch Logs agent, optimizing
+syslog settings, setting up automatic security updates, and more. 
+
+Our Script Modules are contained in the `modules/` folder of different GitHub repos since they often come with the 
+different [Infrastructure Packages](https://blog.gruntwork.io/gruntwork-infrastructure-packages-7434dc77d0b1#.6bwor6wxc) 
+we sell.
```

Comment:
In fact, it's probably worth explaining the type of Apps we have:

1. Binaries. `gruntwork-install` simply copies these into `/usr/local/bin`.
1. Scripts. `gruntwork-install` installs these by running the `modules/install.sh` script in the repo.